seed/campaigns.ts

```diff
@@ -57,3 +61,34 @@ export default async function seedCampaigns(
 
   return campaignIds
 }
+
+async function handleUserCreation(
+  prisma: PrismaClient,
+  existingUsers: User[],
+): Promise<User> {
+  let user = existingUsers.shift()
+  if (!user) {
+    const userData = userFactory()
+    user = await prisma.user.create({
+      data: {
+        ...userData,
+        metaData:
+          userData.metaData !== null ? userData.metaData : Prisma.JsonNull,
+      },
+    })
+  }
+  return user
+}
```

Comment:
Yeah.. services w/ responsibility over one resource, should _not_ be going directly to the ORM to perform transactions w/ other types of resources.

This needs to be a call to the creation method in the `UsersService`, not directly to Prisma.

Reply:
Please ignore me.